ringvax/app.py

```diff
@@ -268,19 +282,11 @@ def app():
             )
 
             st.write(
-                "The following table provides summaries of marginal probabilities regarding detection. Aside from the marginal probability of active detection, these are the observed probabilities that any individual is detected in this manner. The marginal probability of active detection excludes index cases, which are not eligible for active detection."
+                "The following table provides summaries of marginal probabilities regarding detection. Aside from the marginal probability of active detection, these are the observed probabilities that any individual is detected in this manner. The marginal probability of active detection excludes index cases, which are not eligible for active detection, including the index case."
```

Comment:
"The marginal probability of active detection excludes index cases, which are not eligible for active detection, including the index case."

I think this is just a typo? Index case is mentioned twice, once excluded and once inclusively ineligible

Reply:
Good catch

Reply:
I think this also highlights that perhaps what we really want, with respect to some of the discussion in #33, is to have 5 summaries, or maybe 6:
- Pr(index detected) (this is just the passive detection probability, accounting for the delay)
- Pr(non-index detected)
- Pr(non-index detected _actively_)
- Pr(non-index detected _pasively_)
- Pr(any case detected before it's infectious)
  - If 6 summaries, this is two: Pr(index case detected before it's infectious) and Pr(non-index case detected before it's infectious)